src/Twig/MarkdownExtension.php

```diff
@@ -174,4 +177,20 @@ public function formatValue($value)
     {
         return \is_array($value) ? json_encode($value) : $value;
     }
+
+    private function isExcluded(ClassParser $classParser, array $tagsToIgnore = [], array $excluded = []): bool {
+        foreach ($tagsToIgnore as $tagToIgnore) {
+            if ($classParser->hasTag($tagToIgnore)) {
```

Comment:
Duplicate from `AbstractReferencesCommand`, can't it be moved to a trait, for example?

Reply:
I want to improve this part not sure how yet. I think that it'd be best if the Configuration wasn't used at all in this class. I'll work on this on another PR.